geopandas/tests/test_geom_methods.py

```diff
@@ -1579,3 +1579,14 @@ def test_sample_points_pointpats(self, size):
 
         with pytest.raises(AttributeError, match="pointpats.random module has no"):
             gs.sample_points(10, method="nonexistent")
+
+    @pytest.mark.skipif(
+        not (compat.USE_PYGEOS or compat.USE_SHAPELY_20),
+        reason="offset_curve is only implemented for pygeos, not shapely",
+    )
+    def test_offset_curve(self):
+        oc = GeoSeries([self.l1]).offset_curve(1, join_style="mitre")
+        expected = GeoSeries([LineString([[-1, 0], [-1, 2], [1, 2]])])
+        assert_geoseries_equal(expected, oc)
+        assert isinstance(oc, GeoSeries)
+        assert self.g1.crs == oc.crs
```

Comment:
You use `l1` above and `g1` here, I believe incorrectly. 

Also, `l1` does not have any CRS attribute so this check boils down to `None == None`, which does not really check if the CRS is properly preserved.

Reply:
I was going off what was done in the `minimum_bounding_circle` tests which also does the same crs comparison resulting in a None == None assertion.

I'll remove the crs check from this test as it's not adding value